scripts/airmon-ng.linux

```diff
@@ -1616,7 +1618,7 @@ for iface in $(printf "%s" "${iface_list}"); do
 		if [ "${1}" = "stop" ] && [ "${2}" = "${iface}" ]; then
 			stopwlIface "${iface}"
 		fi
-	elif [ "${DRIVER}" = "8723cs" ]; then
+	elif [ "${DEPRECATED}" = "1" ]; then
```

Comment:
Please include the driver check as well, in the event ${DEPRECATED} gets used with other drivers.

Reply:
elif [ "${DEVICEID}" = '0x024c:0xb703' ]; then
  CHIPSET='Realtek RTL8723CS'

in line 988, this will check for the driver also. anything else I need to mention?